### ISSUE TITLE
Allow more than 100 challenges in one milestone

### ISSUE THREAD

**Issue by maxceem** [topcoder-platform/tc-project-service#689]
We have the functionality to create challenges inside milestones in Work Manager:

![image](https://user-images.githubusercontent.com/146016/144831190-cb933c5e-0bb4-447a-ae91-1b7c4ec103b8.png)

Which letter are shown inside Connect App:

![image](https://user-images.githubusercontent.com/146016/144831373-49c443c1-96ed-4d54-a3be-1f4643840424.png)

But if we try to create more than 100 challenges in one milestone we are getting an error:

![image](https://user-images.githubusercontent.com/146016/144831119-6adf7e2c-0762-4463-92c1-5bec178e9213.png)

Are there some reasons to restrict the number of challenges in one milestone?
Could we allow creating any number of challenges in one milestone?

**Comment by vikasrohit** [comment]
There is no specific reason for putting 100 as limit for the number of challenges per phase (milestone). It was only to see what is the practical limit on number of challenges that we can see for a phase/milestone. We never expected more than 100 challenges per phase. We thought, it would make sense to add new phase/milestone in such cases. We can increase this limit to 200 for now or can think of eliminating this limit completely.

**Comment by maxceem** [comment]
So far I'm not sure if we ever would come across with such a limitation on PROD. At the moment we only have this issue during automation testing. When we run test automation in WM it creates challenges in the same milestone. As in WM there is no functionality to add milestones we could not add new milestones in WM automatically.

So to make sure that test would always work we would need to remove this limitation completely. Otherwise during WM testing we would have to make actions in Connect which would not so good, as WM test would depend on changes in Conenct. So changes in Connect might brake tests in WM.

**Comment by vikasrohit** [comment]
Yep, understood. But I guess, if the apps are coupled, there tests would be coupled as well. I mean we need to create milestone first to create challenge. May be we can call projects api instead of using connect app front end, to create milestones.
